ChangeLog

```diff
@@ -3,6 +3,10 @@
    * scripts/abundance-dist.py: disambiguate documentation for force and 
    squash options
 
+2015-04-13 Scott Fay <scott.a.fay@gmail.com>
```

Comment:
At the top of the file, please :-)
Two spaces between your name and the date; two spaces between your name and your bracketed email address. Thanks!